.travis.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Xaptum, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License
+
+sudo: required
+
+language: c
+
+services:
+  - docker
+
+compiler: gcc
+
+jobs:
+  include:
+
+# Release build, gcc
+  - env: TYPE=RELEASE
+    before_script:
+      - docker pull xaptum/xtt-cpp:0.1.0
+
+    script:
+      - docker run -v $(pwd):/xttcpp/ -d -t --name xttcpp xaptum/xtt-cpp:0.1.0
+      - docker exec -it xttcpp bash -c "cd xttcpp && mkdir -p build"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && cmake .. -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && make"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && ctest -E tpm -VV"
+
+# Debug build
+  - env: TYPE=DEBUG
+    before_script:
+      - docker pull xaptum/xtt-cpp:0.1.0
+
+    script:
+      - docker run -v $(pwd):/xttcpp/ -d -t --name xttcpp xaptum/xtt-cpp:0.1.0
+      - docker exec -it xttcpp bash -c "cd xttcpp && mkdir -p build"
+      - docker exec -it xttcpp bash -c "cd xttcpp/build && cmake ../CMakeLists.txt"
```

Comment:
Include the `CMAKE_C[XX]_COMPILER` flags here, too, like above.

Also, since this is a DEBUG build, also include the `CMAKE_BUILD_TYPE=DEBUG`. In fact, for the RELEASE builds, also include `CMAKE_BUILD_TYPE=RELEASE` just to explicitly set that.